.github/workflows/integration.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        test-suite: [TestEthereumE2ESuite, TestFabricE2ESuite]
+        test-suite: [TestEthereumMultipartyE2ESuite, TestEthereumGatewayE2ESuite, TestEthereumMultipartyTokensRemoteNameE2ESuite, TestEthereumNamespaceE2ESuite, TestFabricGatewayE2ESuite, TestFabricMultipartyE2ESuite]
         blockchain-provider: [geth, besu, fabric]
         database-type: [sqlite3, postgres]
         exclude:
```

Comment:
The excluded combos need to be adjusted below for the new suite names. And you definitely need to exclude the combos of "fabric" with any test suite that uses tokens.